src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = function(a, b) {
+    if (b.toString() === a.toString()) {
+      return 0;
+    };
+
+    return a.toString() > b.toString()
+      ? 1
+      : -1;
+  }) {
+    let x = 0;
```

Comment:
Variable 'x' is not named descriptively. Consider using a more descriptive name, such as 'temp' or 'swap' to indicate its purpose.